Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -878,7 +907,9 @@ internal actor DefaultRoomLifecycleManager<Contributor: RoomLifecycleContributor
     /// - Parameters:
     ///   - forcedOperationID: Allows tests to force the operation to have a given ID. In combination with the ``testsOnly_subscribeToOperationWaitEvents`` API, this allows tests to verify that one test-initiated operation is waiting for another test-initiated operation.
     internal func _performReleaseOperation(forcingOperationID forcedOperationID: UUID? = nil) async {
-        await performAnOperation(forcingOperationID: forcedOperationID) { operationID in
+        // See note on performAnOperation for the current need for this force try
+        // swiftlint:disable:next force_try
+        try! await performAnOperation(forcingOperationID: forcedOperationID) { operationID in
```

Comment:
Why forced here?

Reply:
This is a result of the change in b58f9d783211026201c5c6d5ab1ed1361ea874e8 to make `waitForCompletionOfOperationWithID` have a typed throw (i.e. to declare that it throws `ARTErrorInfo` instead of just `Error`); it meant that I had to change `performAnOperation` to always throw `ARTErrorInfo` instead of only throwing an error if the `body` closure throws an error.

Reply:
(There might be a better way of handling all this stuff; I hope so. But this is the best I could do in the limited time I have now.)

Reply:
I think you should just mark `performReleaseOperation...` funcs as throwing then.

Reply:
But it's a minor, just make sure it will be addressed later (maybe create an issue for resolving all `swiftlint:disable:next force_try`)